src/Support_Endpoints/Core.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Core settings listing endpoint for Support Hub
+ */
+
+namespace Avidly\Support_Client\Support_Endpoints;
+
+class Core extends Support_Endpoint {
```

Comment:
I would name the parent class either Endpoint or REST_Endpoint, Support is redundant with the namespace.